charts/base-cluster/templates/weave/gitops.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.weaveworks.enabled }}
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  annotations:
+    metadata.weave.works/description: This is the Weave GitOps Dashboard.  It provides
+      a simple way to get insights into your GitOps workloads.
+  name: weave-dashboard
+  namespace: weave
+spec:
+  chart:
+    spec:
+      chart: weave-gitops
+      sourceRef:
+        kind: HelmRepository
+        name: weaveworks
+  interval: 5m
+  values:
+  {{- .Values.weaveworks | toYaml | nindent 4 }}
```

Comment:
Definitely not like this, we should provide a nice, usable DSL, not just let the user write the plain values, see other comments

Reply:
I just want to limit the values which the user can enter with the values schema but keep the configuration abillity the gitops-chart offers

Reply:
I understand, but we should make it better than the chart

Even just having a plain password instead of having to hash it yourself is worth it

Reply:
reformated